crowdsourcing/crowdsourcing_tutorial.py

```diff
@@ -295,15 +273,16 @@ def encode_text(text):
 sklearn_model = LogisticRegression(solver="liblinear")
 sklearn_model.fit(train_vectors, probs_to_preds(Y_train_prob))
 
+# %%
 print(f"Accuracy of trained model: {sklearn_model.score(test_vectors, Y_test)}")
 
 # %% [markdown]
-# We now have a model with accuracy not much lower than the LabelModel, but with the advantage of being faster and cheaper than crowdsourcing, and applicable to all future examples.
+# We now have a trained model that can be applied to future examples without requiring crowdsourced labels, and with accuracy not much lower than the `LabelModel` that _does_ have access to crowdsourced labels!
 
 # %% [markdown]
 # ## Summary
 #
 # In this tutorial, we accomplished the following:
-# * We showed how Snorkel can handle crowdsourced labels, combining them with other programmatic LFs to improve coverage.
-# * We showed how the LabelModel learns to combine inputs from crowd workers and other LFs by appropriately weighting them to generate high quality probabilistic labels.
-# * We showed that a classifier trained on the combined labels can achieve a fairly high accuracy while also generalizing to new, unseen examples.
+# * We demonstrated how to combine crowdsourced labels with other programmatic LFs to improve coverage.
+# * We used the `LabelModel` to learn how to combine inputs from crowdworkers and other LFs to generate high quality probabilistic labels.
```

Comment:
I don't think we should say 'to learn how'. Just 'We used the `LabelModel` to combine inputs...'